mycroft/skills/phillips_hue/test/intent/sample1.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "turn off the lights",
+  "intent_type": "TurnOffIntent",
+  "intent": {
+    "TurnOffKeyword": "turn off the lights"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab.